test_discovery.py

```diff
@@ -426,6 +426,29 @@ def evaluate_entities(test_dict, resp, verbose):
     return test_single_case(test_dict, observed_entity_dict, resp, verbose)
 
 
+def print_table(timing_list, top_n=5, first_k_chars=25):
+    """ Prints a formatted table ordered by longest test timings. 
+        
+        :param timing_list: namedtuple, list of namedtuples representing testing results
+    """
```

Comment:
The indentation here isn't the same as the docstrings below (`test_all`). We should probably be consistent with our formatting.

Reply:
Agreed we should match https://www.python.org/dev/peps/pep-0257/#handling-docstring-indentation